### ISSUE TITLE
Add multi-value support for parameter 'scanPath'

### ISSUE THREAD

**Issue by HaGGi13** [dependency-check/azuredevops#104]
**AS-IS**
Current it is possible to provide one value for the extension's parameter `scanPath` only. So only one path can be analyzed in one build execution at once only too. Each additional path must be executed in additional build executions.
That's quiet unhandy, if one directory's content is needed to be analyzed and is located in the root, but another one is on the third sub-directory level and both are needed to be part of one single report or result file.
As a an example find followed directory structure, in which _Folder_3_ should be analyzed in complete (incl. sub-folders), but folder _Sub-Folder2.2.1_ too. Thereby, previously mentioned folders/structure must be analyzed at once and be part of the result files/reports.

```
Root
+---Folder_1
|   \---Sub-Folder1.1
+---Folder_2
|   +---Sub-Folder2.1
|   +---Sub-Folder2.2
|   |   +---Sub-Folder2.2.1
|   |   +---Sub-Folder2.2.1
|   |   \---Sub-Folder2.2.1
|   \---Sub-Folder2.3
\---Folder_3
    +---Sub-Folder3.1
    \---Sub-Folder3.2
```

**TO-BE**
It would be worthwhile to have the possibility to be able to use multible values for aforementioned parameter `scanPath` to being able to let multiple path at once analyse. Thereby, the way of usage of parameter `format` could be adopted. So, in concrete, just providing the values by comma (',') separated, because the used/encapsulated [CLI supports this functionality already](https://jeremylong.github.io/DependencyCheck/dependency-check-cli/arguments.html) by providing its parameter `scan` multiple times with the to analyze paths. That means, it's possible to split the comma separated values and add each of them as parameter `scan` as it is implemented for parameter `format`.

**Comment by HaGGi13** [comment]
I have an implementation already, but not tested yet. But feel free to have a look, your feedback is appreciated, even if it's a super small change.
I create a PR as soon as it's tested.

[feature/104-support-multi-scanpath-value](https://github.com/dependency-check/azuredevops/compare/main...HaGGi13:azuredevops:feature/104-support-multi-scanpath-value)

**Comment by ejohn20** [comment]
I have merged this and released in v6.1.0. Please verify that the fix is working as expected.
